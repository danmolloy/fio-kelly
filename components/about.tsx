import React, { useEffect, useRef } from "react";
import IndexSection from "./indexSection";
import Image from "next/image";
import { motion } from "framer-motion";

const paragraphs = [
  "Irish flautist Fiona Kelly has been hailed by the New York Times as a player with “impressive technique and elegant musicianship”.",
  "Based in London, Fiona is in demand as an orchestral and chamber musician. She leads a busy freelance career performing as guest principal flute with many of the UK’s leading orchestras including the London Philharmonic Orchestra, the Philharmonia, Royal Philharmonic Orchestra, the Academy of St Martin in the Fields and the Orchestra of the Royal Opera House, Covent Garden amongst others.",
  "Solo and chamber music engagements have led to collaborations with ensembles and artists including the Wigmore Soloists, Antje Weithaas, Uri Caine and Håkan Hardenberger. Fiona has performed concertos at the BBC Proms, Rheingau and Heidelberg festivals. With a keen interest in new music, Fiona has premiered and recorded new works by Uri Caine, Stephan Mackey and Ian Wilson.",
  "Fiona has a duo with her harpist sister Jean Kelly. They have performed recitals and concertos throughout the U.K. and Ireland. A recent highlight was a recital at the National Concert Hall Chamber Music Series: The Natural World, with a programme titled “Toward the Sea”. The Kelly Sisters often perform works and arrangements of Irish music by their grandfather, composer T.C Kelly.",
  "Fiona enjoys regularly returning home to Ireland to the Irish Chamber Orchestra, where she is guest principal flute. Before relocating to London, Fiona was principal flute of the Swedish Chamber Orchestra from 2012-2016.",
  "Beginning her studies in Cork with her mother Evelyn Grant and Sabine Ducrot, Fiona went on to complete her studies at Trinity Laban Conservatoire with Anna Noakes, and at The Juilliard School with Robert Langevin.",
  "Fiona is a flute professor and currently Interim Head of Woodwind at Trinity Laban Conservatoire. She also gives masterclasses at music departments and colleges throughout the UK and Ireland.",
];

export default function AboutSection() {
  return (
    <IndexSection title={"About"} classNames=" bg-white">
      <div
        data-testid="about-section"
        className=" w-full h-full  flex flex-row "
      >
        <article className="text-black text-base tracking-wider lg:w-2/3 font-sans">
          {paragraphs.map((text, index) => (
            <motion.p
              key={index}
              className="my-2"
              initial={{ opacity: 0 }}
              whileInView={{ opacity: 1 }}
              viewport={{ once: true, amount: 0.2 }}
              transition={{ duration: 0.5, delay: index * 0.1 }}
            >
              {text}
            </motion.p>
          ))}
        </article>
        <div className="hidden lg:inline mx-8">
          <Image
            alt="Grayscale silhouette of Fiona holding her flute"
            width="363"
            height="429"
            src="/images/fk-bw-flute.jpg"
          />
        </div>
      </div>
    </IndexSection>
  );
}
