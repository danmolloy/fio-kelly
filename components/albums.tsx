import Image from "next/image";
import IndexSection from "./indexSection";
import Link from "next/link";
import { useState } from "react";
import { FaSpotify } from "react-icons/fa";


export const albumsArr: {
  title: string
  artists: string
  src: string
  link: string
}[] = [
  {
    title: "The Brandenburg Project",
    artists: "Swedish Chamber Orchestra & Thomas Dausgaard",
    src: "/albums/brandenburg-project.jpg",
    link: "https://open.spotify.com/album/1Qx8aCFUf8I9YSrD89266A?si=Q1Nx2khlT9miP5pvic00fw"
  },
  {
    title: "Folk Songs of the British Isles",
    artists: "Various, including Fiona Kelly & Jean Kelly",
    src: "/albums/folk-songs.jpg",
    link: "https://open.spotify.com/track/5ebx7aiNIlcj6iqnpnh69a?si=DLSo7nlNRSak0dIv6HpCKw"
  },
  {
    title: "Wild is the Wind",
    artists: "Ian Wilson",
    src: "/albums/wild-is-the-wind.jpg",
    link: "https://open.spotify.com/track/2DVIuwdVqu1BQVaODk6tQI?si=Hh1FaK9wSdqFJI__1GKLzA"
  },
  {
    title: "Symphony No. 5: Prokofiev",
    artists: "Santtu-Matias Rouvali & Philharmonia Orchestra",
    src: "/albums/prokofiev-symphoy-5.jpg",
    link: "https://open.spotify.com/album/4qm4iHBsCZul7hSFZQJ0nK?si=cYqWoWw9SZix9qDpz81Fkw"
  },
  {
    title: "A Midsummer Night's Dream",
    artists: "Swedish Chamber Orchestra & Thomas Dausgaard",
    src: "/albums/mendelssohn-midsummer.jpg",
    link: "https://open.spotify.com/album/5BFSaFyBhAifMNJo0HTw7Z?si=Yv3p3QkQTfKJ0hwanhVa7g"
  },
  {
    title: "Brahms Symphony No. 3",
    artists: "Swedish Chamber Orchestra & Thomas Dausgaard",
    src: "/albums/brahms-symphony-3.jpg",
    link: "https://open.spotify.com/track/7jZoI2PHAJzY9RdAiysul6?si=8u6CukG8RaOWpNUZReIkBw"
  },
  {
    title: "Brett Dean: Shadow Music",
    artists: "Swedish Chamber Orchestra & Brett Dean",
    src: "/albums/dean-swedish.jpg",
    link: "https://open.spotify.com/album/0FA63Lj197qGTdI1tQSX1n?si=J2sMyAdxTJOdbRAc2vypvA"
  },
  {
    title: "Mozart: Piano Concertos No. 19 & 27",
    artists: "Francesco Piemontesi, Andrew Manze & Scottish Chamber Orchestra",
    src: "/albums/mozart-piemontesi.jpg",
    link: "https://open.spotify.com/album/5ppe0nPr49exiRuTLW9zQ3?si=s59DWgcWRpCw5KglJqxIOw"
  },
]

export default function Albums() {
  const [selected, setSelected] = useState<number|null>(null)

  return (
    <IndexSection title={"Listen"} classNames={/* selected !== null ? `bg-zinc-200 ease-in-out duration-500 transform` :  */`ease-in-out duration-500 transform bg-gradient-to-b from-zinc-100 to-zinc-100 via-zinc-50`}>
      <div data-testid="albums-section" className=" w-full flex flex-row flex-wrap items-start justify-evenly">
        {albumsArr.map((i, index) => (
          <div  key={i.src} onMouseEnter={() => setSelected(index)} onMouseLeave={() => setSelected(null)} className={` m-1 mx-3 flex flex-col items-center  justify-center `} >


            <Image onClick={() => setSelected(selected === index ? null : index)} className={`shadow-2xl  ${selected !== null && selected !== index && 'bg-zinc-100 opacity-0  ease-in-out duration-500 transform'} self-start ease-in-out duration-500 transform `} src={i.src} width={250} height={250} alt={`Album cover of ${i.title} by ${i.artists}`} title={i.title}/>
            <div className={`${selected === index ? "opacity-100 " : "opacity-0 "} self-start max-w-[250px] p-1`}>
              <p className="flex flex-wrap font-semibold text-base">
                {i.title}
              </p>
              <p className="flex flex-wrap">
                {i.artists}
              </p>
              <Link href={i.link} className="flex flex-row items-center hover:underline">
              <FaSpotify />
              <p className="ml-1">Listen on Spotify</p>
              
              </Link>
            </div>
          </div>
          
        ))}
      </div>
    </IndexSection>
  )
}

{/* <Link data-testid={`${i.link}-link`} target="_blank" href={i.link} className=" " key={i.src}>
          <Image className="shadow-2xl m-4 hover:shadow-zinc-400 active:shadow-zinc-300" src={i.src} width={250} height={250} alt={`Album cover of ${i.title} by ${i.artists}`} title={i.title}/>
        </Link> */}