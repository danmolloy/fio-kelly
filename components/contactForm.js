import styles from '../styles/Home.module.css';
import { Formik, Field, Form, ErrorMessage } from 'formik';
import * as Yup from 'yup';
import { useState } from 'react';
import Link from 'next/link';

export default function ContactForm() {
  const [sendStatus, setSendStatus] = useState(null)

  const sendFail = (<div><h2 className="text-2xl">Message failed to send.</h2> <p>Please <a href='mailto:fiokelly@gmail.com' className='text-blue-500'>send an email</a>.</p></div>)

  const sendSuccess = (<div><h2 className="text-2xl">Message recieved!</h2><p>I will get back to you as soon as possible.</p></div>)

  const sendingMsg = (<div><h2 className="text-2xl">Message sending...</h2></div>)

  return (
    <Formik    
    initialValues={{
      name: '',
      email: '',
      message: '',
    }}
    validationSchema={Yup.object({
      name: Yup.string()
        .max(30, 'Must be 30 characters or less')
        .required('Required'),
      email: Yup.string()
        .email('Invalid email address')
        .required('Required'),
      message: Yup.string()
        .max(200, 'Must be 200 characters or less')
        .required('Required')
    })}
    onSubmit={async (values, actions ) => {
      console.log('Sending')
      setSendStatus(sendingMsg)
      await new Promise(resolve => setTimeout(resolve, 1000))
        
      fetch('/api/contact', {
        method: 'POST',
        headers: {
          'Accept': 'application/json, text/plain, */*',
          'Content-Type': 'application/json'
        },
        body: JSON.stringify(values)
      }).then((res) => {
        console.log('Response received')

        if (res.status === 200) {
          setSendStatus(sendSuccess)
          actions.setSubmitting(false)
          actions.resetForm()
        } else {
          setSendStatus(sendFail)
        }
        })
      }}> 

      <Form className="form">
      <h1 className="page-title self-start">Contact</h1> 
          <div className='contact-blurb'>
          <p className='my-1'>For concert bookings, please contact <Link href={"https://maslink.co.uk/"}><a className='link'>The Musicians Answering Service</a></Link>.</p>
          <p className='my-1'>For all other enquiries, complete the form below.</p>
          </div>

          <label htmlFor='name-input' className="form-label">Name</label>
          <Field 
            id="name-input" 
            className="form-input"
            type="text"
            name="name"/>
          <ErrorMessage name="name">
            { msg => <div className="form-error">{msg}</div> }
          </ErrorMessage>

          <label htmlFor='email-input' className="form-label">Email</label>
          <Field
            id="email" 
            className="form-input" 
            type="email"
            name="email"/>
          <ErrorMessage name="email">
          { msg => <div className="form-error">{msg}</div> }
          </ErrorMessage>

          <label htmlFor='msg-text' className="form-label">Message</label>
          <Field 
            id="msg-text" 
            className="text-input" 
            type="text"
            name="message"/>
          <ErrorMessage name="message">
            { msg => <div className="form-error">{msg}</div> }
          </ErrorMessage>
        <button id="submit-button" type='submit' className="submit-btn">Submit</button>
        <div>
        {sendStatus}
      </div>
      </Form> 
    </Formik>
  )
}