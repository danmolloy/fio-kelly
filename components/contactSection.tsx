import IndexSection from "./indexSection";
import { Formik, Field, Form, ErrorMessage } from 'formik';
import * as Yup from 'yup';
import { useState } from 'react';
import Image from "next/image";
import axios from "axios";

export default function ContactSection() {
  const [sendStatus, setSendStatus] = useState<"success"|"sending"|"err"|null>(null)

  const sendFail = (<div role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc"><h2 id="dialog-title" className="text-lg">Message failed to send.</h2> <p id="dialog-desc" className="text-sm">Please <a href='mailto:fiokelly@gmail.com' className='text-blue-500'>send an email</a>.</p></div>)

  const sendSuccess = (<div role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc"><h2 id="dialog-title" className="text-lg">Message recieved!</h2><p id="dialog-desc" className="text-sm">I will get back to you as soon as possible.</p></div>)

  const sendingMsg = (<div role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc"><h2 id="dialog-title" className="text-lg">Message sending...</h2><p id="dialog-desc" className="text-sm">Please wait.</p></div>)

  return (
    <IndexSection title="Contact" classNames="bg-gradient-to-b from-zinc-100 to-zinc-100 via-zinc-50">
      <div data-testid="contact-form" className="flex flex-row w-full justify-between  h-[30rem]">
    <Formik    
    initialValues={{
      name: '',
      email: '',
      message: '',
    }}
    validationSchema={Yup.object({
      name: Yup.string()
        .max(30, 'Must be 30 characters or less')
        .required('name required'),
      email: Yup.string()
        .email('Invalid email address')
        .required('email required'),
      message: Yup.string()
        .max(500, 'Must be 200 characters or less')
        .required('message required')
    })}
    onSubmit={async (values, actions ) => {

      setSendStatus("sending")
      await new Promise(resolve => setTimeout(resolve, 1000))
      axios.post("/contact", values)
      .then(() => {
        setSendStatus("success")
        actions.resetForm()
      })
      .catch((error) => {
        setSendStatus("err")
        const errorMessage =
          error.response.data.error || 'An unexpected error occurred.';
        actions.setStatus(errorMessage);
      })
      .finally(() => {
        actions.setSubmitting(false);
      });
      
        
      }}> 
      {(props) => (
      <Form className="font-sans flex flex-col lg:w-1/2  ">
          <div className=' mb-2 '>
          <p className='my-1'>For concert bookings, please contact <a target="_blank" href={"https://maslink.co.uk/client-directory?client=KELLF1&instrument=FLUTE1"} className="hover:text-blue-500 underline">The Musicians Answering Service</a>.</p>

          <p className='my-1'>For all other enquiries, complete the form below.</p>
          </div>
         <div className="flex flex-col my-1">
          <label htmlFor='name-input' className="form-label">Name</label>
          <Field 
            id="name-input" 
            className="text-black border border-black rounded-md w-full p-1"
            type="text"
            name="name"/>
            <div className="h-6">
          <ErrorMessage name="name">
            { msg => <div className="text-sm text-red-500 h-6">{msg}</div> }
          </ErrorMessage>
          </div>
          </div>
          <div className="flex flex-col my-1">
          <label htmlFor='email-input' className="form-label">Email</label>
          <Field
            id="email-input" 
            className="text-black border border-black rounded-md w-full p-1" 
            type="email"
            name="email"/>
            <div className="h-6">
          <ErrorMessage name="email" >
          { msg => <div className="text-sm text-red-500 h-6">{msg}</div> }
          </ErrorMessage>
          </div>
          </div>
          <div className="flex flex-col my-1 mb-0">
          <label htmlFor='msg-text' className="form-label">Message</label>
          <Field 
          multiline="6"
          maxLength="500"
          rows="4"
          component="textarea"
            id="msg-text" 
            className=" text-black border border-black rounded-md w-full p-1"
            type="textarea"
            name="message"/>
            <div className="h-6">
            {props.values.message.length > 0 && <p className='self-center text-sm mx-2 opacity-40'>{`${props.values.message.length}/500`}</p>}

          <ErrorMessage name="message">
            { msg => <div className="text-sm text-red-500">{msg}</div> }
          </ErrorMessage>
          </div>
          </div>
        <button disabled={props.isSubmitting || (sendStatus === "sending")} id="submit-button" type='submit' className="disabled:opacity-40 disabled:hover:cursor-default hover:bg-blue-50 border shadow-sm border-blue-500 text-blue-500 m-2 p-1 w-24 rounded self-end">Submit</button>
        <div>
        {sendStatus === "sending" 
        ? sendingMsg
        : sendStatus === "err" 
        ? sendFail
        : sendStatus === "success" 
        ? sendSuccess
        : null}
      </div> 
      </Form> )}
    </Formik>
    <div className="hidden lg:flex mx-12">
      <Image alt="Fiona Kelly" src={"/images/fiona-profile.jpg"} width="325" height="425" />
    </div>
    </div>
    </IndexSection>
  )
}