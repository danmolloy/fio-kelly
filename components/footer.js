import Link from 'next/link'
import { BsTwitter } from 'react-icons/bs'

export default function Footer() {
  return (
    <div className='footer'>
      <Link href="https://twitter.com/fizzlekelly">
        <div className='hover:text-blue-500'>
          <BsTwitter className='icons'/>
        </div>
      </Link>
    </div>
  )
}