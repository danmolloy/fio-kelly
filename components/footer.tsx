import { animateScroll as scroll } from 'react-scroll'
import { AiOutlineTwitter } from 'react-icons/ai'

export default function Footer() {
  return (
    <div className='z-10 h-24 bottom-0 flex flex-row items-center justify-center border-t  bg-zinc-200'>
      <div className='flex flex-col items-center justify-center'>
      <a target="_blank" href='https://twitter.com/fizzlekelly' className=' text-2xl twitter-blue p-1 hover:text-blue-500 hover:cursor-pointer active:text-blue-400'>
        <AiOutlineTwitter />
      </a>
      <button className='hover:underline font-display text-sm font-light hover:cursor-pointer active:text-zinc-400' onClick={() => scroll.scrollToTop({ smooth: true, duration: 750 })}>
        Back to top
      </button>
      <a className='text-sm font-light font-display self-end mt-3 text-gray-500 underline hover:text-blue-500' href="https://www.purpleraspberry.uk/" target='_blank'>Purple Rasberry Photography</a>

      </div>
    </div>
  )
}