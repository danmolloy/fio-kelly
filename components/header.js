import Link from 'next/link'
import { useEffect, useState } from 'react'
import { AiOutlineMenu } from 'react-icons/ai'
import MenuIcon from './menuIcon'

export default function Header({showMenu, setShowMenu, home}) {
  //const [scroll, setScroll] = useState('')


 /*  useEffect(() => {
    window.addEventListener('scroll', e => {
      setScroll(document.scrollingElement.scrollTop)
    })
  }) */


  return (
    <div className="header">
    <Link href="/">
        <a>
        <h2 className='header-name strokeme'>FIONA KELLY</h2>
        </a> 
      </Link>
      <MenuIcon setShowMenu={() => setShowMenu()} showMenu={showMenu}/>
    </div>
  )
}