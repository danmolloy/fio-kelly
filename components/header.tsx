import { useEffect, useState } from 'react'
import MenuIcon from './menuIcon'
import { Link, animateScroll as scroll } from 'react-scroll'

export const menuItems: {title: string}[] = [
  {
    title: "About",
  },
  {
    title: "Listen",
  },
  {
    title: "Gallery",
  },
  {
    title: "Contact",
  },
]

export type HeaderProps = {
  showMenu: boolean
  setShowMenu: () => void
  setShowHeader: (arg: boolean) => void
  showHeader: boolean
}

export default function Header(props: HeaderProps) {
  const {showMenu, setShowMenu, setShowHeader, showHeader} = props;
  const [scrollPosition, setScrollPosition] = useState(0);

  useEffect(() => {
    function handleScroll() {
      const currentPosition = window.scrollY;
      if (currentPosition > scrollPosition + 100) {
        setScrollPosition(currentPosition);
        setShowHeader(false)
      } else if (currentPosition < scrollPosition - 5) {
        setScrollPosition(currentPosition);
        setShowHeader(true)
      }
    }

    window.addEventListener('scroll', handleScroll);

    return () => {
      window.removeEventListener('scroll', handleScroll);
    };
  }, [scrollPosition]);


  return (
    <div
    data-testid="page-header" 
    className={
      showMenu 
      ?      "font-display  transition duration-500 ease-out fixed fill-white  z-30 h-16 w-screen top-0 flex flex-row  items-center justify-between   text-white "

      : !showHeader && scrollPosition < 750
      ? "font-display -translate-y-full transition duration-1000 fill-white fixed z-30 h-16 w-screen top-0 flex flex-row  items-center justify-between  text-zinc-100"  
      : !showHeader
      ? "font-display -translate-y-full transition duration-1000 bg-white fixed z-30 h-16 w-screen top-0 flex flex-row items-center justify-between shadow fill-black text-black"  
      : scrollPosition < 750
      ? "font-display  transition duration-500 ease-out fixed fill-white  z-30 h-16 w-screen top-0 flex flex-row  items-center justify-between   text-zinc-100"
      : "font-display transition duration-500 ease-out fixed  bg-white z-30 h-16 w-screen top-0 flex flex-row  items-center justify-between shadow fill-black text-black "}>
      <h1 className={'md:hidden p-4 '}>Fiona Kelly</h1>
      <div className='hidden md:flex w-screen  flex-row  justify-end pr-4'>
        <button className='my-6 p-2 text-lg font-light hover:cursor-pointer active:text-zinc-400' onClick={() => scroll.scrollToTop({ smooth: true, duration: 500 })}>
          Home
        </button>
      {menuItems.map(i => (
        <Link href='/' data-testid={i.title} onClick={() => setTimeout(() => setShowHeader(false), 750)} activeClass="active" to={i.title.toLowerCase()} spy={true} smooth={true} offset={50} duration={500}  key={i.title} className=' p-4 m-2 text-lg font-light hover:cursor-pointer active:text-zinc-400'>
          <button className='p-2'>{i.title}</button>
        </Link>
      ))}
      </div>
      <MenuIcon setShowMenu={() => setShowMenu()} showMenu={showMenu}/>
    </div>
  )
}