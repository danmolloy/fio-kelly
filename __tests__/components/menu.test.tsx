import "@testing-library/jest-dom"
import { act, fireEvent, render, screen } from "@testing-library/react"
import Menu, { MenuProps } from "../../components/menu"
import { menuItems } from "../../components/header"
import { Link, animateScroll as scroll } from 'react-scroll'

jest.mock("react-scroll", () => ({
  animateScroll: {
    scrollToTop: jest.fn()
  },
  Link: (props) => <div>{props.children}</div>
}))


const mockProps: MenuProps = {
  setShowHeader: jest.fn(),
  setShowMenu: jest.fn(),
  showMenu: true
}

describe("<Menu />", () => {
  beforeEach(() => {
    render(<Menu {...mockProps} />)
  })
  
  it("menu-div is in the document", () => {
    const menuDiv = screen.getByTestId("menu-div")
    expect(menuDiv).toBeInTheDocument()
  })
  it("menuItems are in the document & call setShowMenu(false) on click", () => {
    for (let i = 0; i < menuItems.length; i ++) {
      const menuItem = screen.getByText(menuItems[i].title)
      expect(menuItem).toBeInTheDocument()
    }
  })
  it("Home button is in the document and calls setShowMenu(false) & scrollToTop() on click", () => {
    const homeBtn = screen.getByText("Home")
    expect(homeBtn).toBeInTheDocument()
    act(() => {
      fireEvent.click(homeBtn)
    })
    expect(mockProps.setShowMenu).toHaveBeenCalledWith(false)
    expect(scroll.scrollToTop).toHaveBeenCalledWith({
      smooth: true, 
      duration: 500
    })
  })
  it("matches snapshot", () => {
    const menuDiv = screen.getByTestId("menu-div")
    expect(menuDiv).toMatchSnapshot()
  })
})