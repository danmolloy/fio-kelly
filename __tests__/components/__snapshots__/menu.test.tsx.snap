// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> matches snapshot 1`] = `
<div
  class="opacity-90 ease-out transition duration-300 rounded-bl flex flex-col z-20 bg-white fixed w-screen  min-h-screen  right-0 shadow text-2xl"
  data-testid="menu-div"
>
  <div
    class="flex flex-col w-full justify-evenly font-display mt-16"
  >
    <button
      class="py-4 m-4 flex flex-col text-black  items-center justify-center hover:text-amber-500 active:text-amber-400"
    >
      Home
    </button>
    <div>
      <div
        class="py-4 m-4 flex flex-col text-black  items-center justify-center hover:cursor-pointer hover:text-amber-500 active:text-amber-400"
      >
        About
      </div>
    </div>
    <div>
      <div
        class="py-4 m-4 flex flex-col text-black  items-center justify-center hover:cursor-pointer hover:text-amber-500 active:text-amber-400"
      >
        Listen
      </div>
    </div>
    <div>
      <div
        class="py-4 m-4 flex flex-col text-black  items-center justify-center hover:cursor-pointer hover:text-amber-500 active:text-amber-400"
      >
        Gallery
      </div>
    </div>
    <div>
      <div
        class="py-4 m-4 flex flex-col text-black  items-center justify-center hover:cursor-pointer hover:text-amber-500 active:text-amber-400"
      >
        Contact
      </div>
    </div>
  </div>
</div>
`;
